homeassistant/components/velbus/light.py

```diff
@@ -80,10 +80,15 @@ def turn_on(self, **kwargs):
                 attr, *args = "set_led_state", self._channel, "on"
         else:
             if ATTR_BRIGHTNESS in kwargs:
+                # Make sure a low but non-zero value is not rounded down to zero
+                if 0 < kwargs[ATTR_BRIGHTNESS] < 3:
```

Comment:
I think it's better if we make 0 the special case and cap all other cases with a minimum of 1. Then we don't need to care about how the conversion calculation is done to know how the check should be written. 

```py
if kwargs[ATTR_BRIGHTNESS] == 0
    brightness = 0
else:
    brightness = max(int((kwargs[ATTR_BRIGHTNESS] * 100) / 255), 1)
```

Reply:
Fixed in commit [b0069aa](https://github.com/home-assistant/core/pull/33139/commits/b0069aaf04a1cbcaac3092861eaac85d99bfb91b)